recipes/polygon-geohasher/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "polygon-geohasher" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5c9f8788fa38df1606c7bd9e85fce636c8520a19a675c106ee387c0f613697ef
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+    - python-geohash
+    - shapely
+
+test:
+  imports:
+    - polygon_geohasher
+    - tests
+
+about:
+  home: "https://github.com/Bonsanto/polygon-geohasher"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "Wrapper over Shapely that returns the set of geohashes that form a Polygon"
+  doc_url: "https://pypi.org/project/polygon-geohasher/"
+  dev_url: "https://github.com/Bonsanto/polygon-geohasher"
+
+extra:
+  recipe-maintainers:
+    - mwilson8
+    - rluria14
```

Comment:
@rluria14 , do you agree in maintain this recipe as well?

Reply:
I agree